runtime/src/main/cpp/Worker.cpp

```diff
@@ -62,10 +62,11 @@ enum {
 THREAD_LOCAL_VARIABLE KInt g_currentWorkerId = 0;
 
 KNativePtr transfer(ObjHolder* holder, KInt mode) {
+  holder->hold();
```

Comment:
Why not just
```
void* result = CreateStablePointer(holder.obj())
holder.clear()
if (!ClearSubgraphReferences(holder->obj(), mode == CHECKED)) {
  ThrowWorkerInvalidState();
}
return result;
```

Technically almost the same, but avoids leaking abstractions and pairs with `AdoptStablePointer` semantically.